examples/demo/site/modules/auth/mail_templates/account_activation.html

```diff
@@ -8,11 +8,7 @@
 	</head>
 
 	<body>
-		<p>Thank you for registering at <a href="$host">ROC CMS</a></p>
-
-		<p>To complete your registration, please click on this link to activate your account:<p>
-
-		<p><a href="$link">$link</a></p>
-		<p>Thank you for joining us.</p>
+		<p>Thank you for applying to  <a href="$host">ROC CMS</a> $user</p>
+		<p>We will review your application and send you a resolution<p>
```

Comment:
So this is not possible anymore for the user to activate himself its account?
Maybe this could be an option ... "Allow account activation by user" .

And then, it is up to the admin to allow or not.
What is your feeling on that?